paket.dependencies

```diff
@@ -9,5 +9,5 @@ nuget FsCheck.Xunit
 nuget xunit.runner.console
 nuget xunit.assert
 nuget FAKE
-nuget Mono.Cecil >= 0.10.0-beta1
```

Comment:
For some reason, the rewriter ceased working when Paket automatically updated the lock file when CommandLineParser was installed. Updating the Mono.Cecil version fixed this.